### PR TITLE
WIP: Add a projectOverrides argument to the skeleton's default.nix(?)

### PR DESCRIPTION
**Pull request by cgibbard**
(also getTextConfig, and this PR includes the ob doc command one)

### CODE REVIEW

File: lib/executable-config/lookup/src/Obelisk/Configs.hs [hunk 1]
```diff
@@ -50,6 +51,9 @@ class Monad m => HasConfigs m where
 instance Monad m => HasConfigs (ConfigsT m) where
   getConfigs = ConfigsT ask
 
+getTextConfig :: HasConfigs m => Text -> m (Maybe Text)
+getTextConfig k = fmap T.decodeUtf8 <$> getConfig k
```
Comment (tomsmalley) [inline lib/executable-config/lookup/src/Obelisk/Configs.hs]: Should this be `decodeUtf8With lenientDecode`?

Comment (cgibbard) [reply lib/executable-config/lookup/src/Obelisk/Configs.hs]: You probably want the backend to fail in the case that its config can't be read, though perhaps we should reflect the encoding failure somehow, I don't know.

Comment (3noch) [reply lib/executable-config/lookup/src/Obelisk/Configs.hs]: In that case we should probably use `fromMaybe (error $ "Failed to decode " <> T.unpack k) . decodeUtf8'`

Comment (Ericson2314) [reply lib/executable-config/lookup/src/Obelisk/Configs.hs]: @3noch agreed, except I think it better to fail in the monad.

File: skeleton/default.nix [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-  # Uncomment and set this to `true` to indicate your acceptance:
-  # config.android_sdk.accept_license = false;
+    system = builtins.currentSystem;
+    iosSdkVersion = "10.2";
+    # You must accept the Android Software Development Kit License Agreement at
+    # https://developer.android.com/studio/terms in order to build Android apps.
+    # Uncomment and set this to `true` to indicate your acceptance:
+    # config.android_sdk.accept_license = false;
   }
+, projectOverrides ? {}
```
Comment (3noch) [inline skeleton/default.nix]: Why encode this as part of the skeleton? Do you see this being used by the average obelisk user? It's easy enough to do yourself you really want it...

**Comment by 3noch** [comment]
We'll revisit this if it becomes a problem...

**Pull request closed without merging**